src/cargo.ts

```diff
@@ -36,7 +50,21 @@ export function getManifestPath(manifestPath: string): string {
   return manifestPathResult;
 }
 
+function isValidUrl(s: string): boolean {
```

Comment:
this check only works for `https://..` and `ssh://...` urls, but not for GitHub style ssh urls `git@github.com:...`, right? I'm ok with that, but it might be worth documenting it. It might also be good to support those urls.

Reply:
Sure thing! Addressed in [765c6da](https://github.com/cargo-lambda/cargo-lambda-cdk/pull/57/commits/765c6dac9419caddaa628712a7e7f0b6cd34c633) and dropping one more dependency :)